ci/test-stable.sh

```diff
@@ -79,6 +80,12 @@ test-stable-perf)
   _ cargo +"$rust_stable" build --all --tests --bins ${V:+--verbose} --features="$ROOT_FEATURES"
   _ cargo +"$rust_stable" test --manifest-path=core/Cargo.toml ${V:+--verbose} --features="$ROOT_FEATURES" -- --nocapture
   ;;
+test-local-cluster)
+  echo "Executing $testName"
+  _ cargo +"$rust_stable" build --all --release --tests --bins ${V:+--verbose}
+  _ cargo +"$rust_stable" test --release --manifest-path=local_cluster/Cargo.toml  ${V:+--verbose} -- --nocapture
```

Comment:
Hm this doesn't look right. --package Solana-local-cluster is how you resolve which package to test. Won't this just run cargo test in release? Also local cluster isn't the only thing that actually runs a local cluster. There are other modules too. Need to exclude all of these from the regular stable run and explicitly run them here?

Oh also remove the echo?

Reply:
the `--manifest-path=local_cluster/Cargo.toml` should run just the local cluster tests. I changed the stable run to use `--manifest-path=core/Cargo.toml` so it shouldn't run the local cluster tests. The echo seems common across all the different entries in this case statement.

Reply:
Ab I clearly don't understand how the manifest path stuff works. We added it to stable perf because the Cuda feature wasn't working without it. That's the only reason. I guess it had a side effect of skipping all the other tests lol.

Okay so the local cluster run should then use multiple manifests? Because now no test in the other modules are running right? Like metrics has a local cluster test. Bench exchange and bench tps use a local cluster. Just make sure they're being run?

Reply:
hmmm good point, I missed the other tests that were using local cluster outside of local_cluster :P. 

What if I refactor the remaining local cluster tests out of their respective modules and into the local cluster crate? Then I flag all those tests with a feature so they don't run in  the `stable` job, but will run when I pass the feature to the `local-cluster` job.

Reply:
I don't know about the feature thing. It just sounds so icky. Sorry I'm not trying to make your life harder xD 

You could run `cargo test --all --exclude solana-local-cluster --exclude other-local-cluster ...` and then when you want to run just those packges your can use `cargo test --package solana-local-cluster --package other-local-cluster ...`

Reply:
hmmm worried that the packages I'm excluding might also have non local-cluster tests that we want to run with regular debug :P, really wish this: https://users.rust-lang.org/t/cargo-build-dependencies-in-release-current-crate-in-debug/20997 was a part of stable

Reply:
Yeah that would be ideal. 
As for non-local-cluster tests, nbd as long as they are run somewhere right? :S